packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -84,8 +92,28 @@ const DEFAULT_VALUES = {
     slackTargets: [] as string[],
     filters: undefined,
     customViewportWidth: undefined,
+    thresholds: [],
 };
 
+const DEFAULT_VALUES_ALERT = {
+    ...DEFAULT_VALUES,
+    cron: '0 10 * * *',
+    thresholds: [
+        {
+            fieldId: '',
+            operator: ThresoldOperator.GREATER_THAN,
+            value: 0,
+        },
+    ],
+};
+
+const thresholdOperatorOptions = [
+    { label: 'Greater than', value: ThresoldOperator.GREATER_THAN },
```

Comment:
you could make this dynamic by iterating on ThresholdOperator enum,and using friendlyName for the label. This works too though.

Reply:
I thought about it, but since those are the values that get stored in the DB, I don't love to tie them so directly to the UI. If we want to change what the display text is, it would break existing saved things OR you'd have to do this anyway. 

It already came up, actually 😄. We changed this to 'is greater than'. If it had been using friendly name I would have had to change the type values. 

I'll leave this for now and if you disagree or theres something I'm not considering, let me know and I wont argue further.